index.html

```diff
@@ -0,0 +1,156 @@
+<!doctype html>
+<head>
+  <title>Static Web HTML and CSS Quiz</title>
+  <link rel="stylesheet" type="text/css" href="quiz.css">
+</head>
+
+<body>
+
+
+<!-- 1st CARD --> 
+
+  <div class="card">
+
+    <header><h1 id=cardTitle>Yellow Kittens</h1></header>
+
+    <section>
+      <!-- The first section should contain three child block elements. -->
+      <img src="http://placekitten.com/g/400/400" class="product_image" alt="Yellow Kittens">
+
+      <!-- The second element contains the product description. -->
+      <p class="product_description">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+      <!-- The third element contains the product availability (e.g. "Available" or "Not Available") -->
+      <div id="availability">Available until 12/31/2020.</div>
+
+    </section> 
+
+    <section>
+      <!-- The second section contains product specifications. -->
+
+      <header><h2>Specifications</h2></header>
+        
+      <!-- The first block element specifies the size. -->
+      <div class="specs">Size: 4" x 4"</div>
+
+        <!-- The second block element specifies the weight. -->
+      <div class="specs">Weight: 8 oz.</div>
+
+      <footer class="specsFooter">Specifications valid until 4/1/2016.</footer>
+    
+    </section>
+
+    <section>
+      <!-- The third section contains the product pricing. -->
+
+      <header><h2>Pricing</h2></header>
+
+      <div class="price">1: $1.99</div>
+      <div class="price">2-50: $1.49</div>
+      <div class="price">51+: $0.99"</div>
+
+    </section>
+
+  </div>
+
+  <footer></footer>
```

Comment:
The empty `footer` isn't hurting anything, but having empty elements that do nothing might end up causing future CSS headaches if you don't watchout